apps/src/templates/SchoolDataInputs.jsx

```diff
@@ -29,8 +29,13 @@ export default function SchoolDataInputs({
   const [country, setCountry] = useState('');
   const [zipSearchReady, setZipSearchReady] = useState(false);
 
-  let COUNTRY_ITEMS = [{value: 'selectCountry', text: i18n.selectCountry()}];
-  for (const item of Object.values(COUNTRIES)) {
+  let COUNTRY_ITEMS = [
+    {value: 'selectCountry', text: i18n.selectCountry()},
+    {value: 'US', text: 'United States'},
+  ];
+  for (const item of Object.values(COUNTRIES).filter(function (item) {
+    return item.label !== 'US';
+  })) {
```

Comment:
Nit: Could the filter logic either be moved to the line before or could the words be changed from "item"? It's a bit confusing with two different `item`s being used

Reply:
I can definitely rename one! Is it less confusing to name the second one 'country' even though both 'items' represent the same thing?

Reply:
Curious what Turner thinks but it might be clearer if the filter logic is on a separate line. IMO if you do that, using the same variable for both would be reasonable.

So something like 
```
nonUsCountries = Object.values(COUNTRIES).filter(item => item.value !== 'US')
for (const item of nonUsCountries) {
....
}
```

Reply:
Ahh clutch okay I updated it. I also added the translation and a couple comments to hopefully make it all clearer!